common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/UserChoiceDialog.java

```diff
@@ -36,15 +36,15 @@
 public class UserChoiceDialog extends SmartcardDialog {
 
     private final PositiveButtonListener mPositiveButtonListener;
-    private final CancelCbaCallback mCancelCbaCallback;
+    private final ICancelCbaCallback mCancelCbaCallback;
 
     /**
      * Creates new instance of SmartcardDialog.
      *
      * @param activity Host activity.
      */
     public UserChoiceDialog(@NonNull final PositiveButtonListener positiveButtonListener,
-                            @NonNull final CancelCbaCallback cancelCbaCallback,
+                            @NonNull final ICancelCbaCallback cancelCbaCallback,
```

Comment:
Since we don't cancel from this dialog anymore, do we still need to pass the cancel callback? I don't see it used in this class anymore.

Reply:
I think same idea applies to SmartcardPromptDialog

Reply:
You can't see it in this PR file without expanding down a bit, but cancelCbaCallback is set to the class variable mCancelCbaCallback. The onCancel() method of it is called when the user presses the cancel button on these particular dialogs that have a cancel button.